docs/application/native/guides/app-management/efl-ui-app.md

```diff
@@ -262,4 +262,4 @@ An application with a background running capability must declare the background
 ```
 The `<metadata key="http://tizen.org/metadata/bacgkround-category/<value>"/>` element has no effect on Tizen 2.3 devices, but on Tizen 2.4 and higher devices, it has the same effect as the `<background-category>` element.
 
-The background category of your application can be specified in the [application project settings](../../../../org.tizen.training/html/native/process/setting_properties_n.htm#manifest) in the Tizen Studio.
+The background category of your application can be specified in the [application project settings](../../../../tutorials/process/setting-properties.md#manifest) in the Tizen Studio.
```

Comment:
../../../../tutorials/process/setting-properties.md#manifest
=> ../../ looks correct. Also the file name is setting-properties-n.md now.

Reply:
The name is changed on PR #130.
 
https://github.com/Samsung/tizen-docs/pull/130/commits/d652a344e0b73aec8d7f862e1ff9acaeea1c9985#diff-963d908b28a05830f351dd5bfb091788